### PR TITLE
Miscellaneous changes

### PR DESCRIPTION
**Pull request by oianmol**
1. Updated Dependency versions
2. Fix Android build because of duplicate kotlin java files because of protobuf-kotlin-lite
3. Fixed Streaming calls to return non suspending flows.

### CODE REVIEW

**Comment by TimOrtel** [comment]
In which scenario do you experience the duplicate classes? The test module, for example, works without issues of duplicate classes? Can you maybe give a project structure where this is an issue?

File: plugin/src/main/java/io/github/timortel/kotlin_multiplatform_grpc_plugin/generate_mulitplatform_sources/generators/service/ServiceWriter.kt [hunk 1]
```diff
@@ -45,25 +45,45 @@ abstract class ServiceWriter(private val isActual: Boolean) {
                             val returnType = if (rpc.isResponseStream) {
                                 Flow::class.asTypeName().parameterizedBy(rpc.response.commonType)
                             } else rpc.response.commonType
+                            if (rpc.isResponseStream) {
```
[the remark this answers is not in the document]
Comment (Anmol92verma) [reply plugin/src/main/java/io/github/timortel/kotlin_multiplatform_grpc_plugin/generate_mulitplatform_sources/generators/service/ServiceWriter.kt]: resolved.

**Comment by TimOrtel** [comment]
I think this PR is too broad and should be split up in multiple PRs each with a specified topic. The most important part of your PR is the fixes to the iOS rpc implementation. Could you create a specific PR for this maybe?

**Comment by TimOrtel** [comment]
The callback flow approach is now used on both ios and js on the develop branch. Thank you for the hint.

**Pull request closed without merging**